CMakeLists.txt

```diff
@@ -0,0 +1,658 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+# Ubuntu 22.04 LTS Jammy Jellyfish, https://wiki.ubuntu.com/Releases, EOSS in June 2027:
+#  - CMake 3.22.1, https://packages.ubuntu.com/jammy/cmake
+#
+# Centos Stream 9, EOL in May 2027:
+#  - CMake 3.26.5, https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/
+cmake_minimum_required(VERSION 3.22)
```

Comment:
if we update the above, this can likely also be bumped, right?

Reply:
We still want to allow the building of Bitcoin Core on Ubuntu 22.04 using the [`cmake`](https://packages.ubuntu.com/jammy/cmake) package available in the system repository.

For reference, the last time the CMake minimum required version was updated was in https://github.com/hebasto/bitcoin/pull/123.